bundle/src/main/java/com/adobe/acs/commons/users/impl/AbstractAuthorizable.java

```diff
@@ -0,0 +1,105 @@
+/*
```

Comment:
copyright across several files are malformatted.

Reply:
I'll check this.  I tend to habitually reformat my code as I work, so possible my formatter munged them up.